.github/workflows/build.yml

```
@@ -61,9 +61,10 @@ jobs:
 
       - uses: cachix/cachix-action@0fc020193b5a1fa3ac4575aa3a7d3aa6a35435ad # v16
         with:
-          # This cache is for the nixpkgs repo checks and should not be trusted or used elsewhere.
-          name: nixpkgs-ci
-          authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
+          # The nixpkgs-ci cache should not be trusted or used outside of Nixpkgs CI.
+          name: ${{ vars.CACHIX_NAME || 'nixpkgs-ci' }}
```

```suggestion
          name: ${{ vars.CI_CACHIX_NAME || 'nixpkgs-ci' }}
```

?

EDIT: we already have `secrets.CACHIX_AUTH_TOKEN`, so I guess it's better to be consistent and use `vars.CACHIX_NAME`.

---

Also: is it worth adding the variable to the main nixpkgs repo, so that we don't need `||` here?